### DISCUSSION TITLE
Please delete this.

### DISCUSSION THREAD

**Discussion by David7c8** [chromiecraft/chromiecraft#2642] [General]

**Comment by David7c8** [comment]
I just want to add that in the thumbnail I am flagged for PVP. I believe it to be a bug. It was the second occurrence and I suspect triggered by my own faction! Maybe it is a feature that allows it but it is concerning. Please don't ban me. 😭 

I'll try and remember to log it the next time it happens.

**Reply by cameron008** [reply]
Were you flagged from the start, or did you get flagged later? If you got flagged later, it might be because you interacted with a flagged player, as in buffing, dueling, healing.

**Reply by David7c8** [reply]
I did give the warrior Arcane Intellect. You are my new best friend.

The first occurrence must have been by doing the same in Ratchet when the only other player there was an ally Troll. Thank you!

**Comment by David7c8** [comment]
I was hoping I could get this post deleted. I released this script far too early. I want it to be a masterpiece with more GUI to set hotkeys, Copy WTF, ini files, the works. And a better video upload.
